plat/kvm/Makefile.uk

```diff
@@ -24,6 +23,7 @@ LIBKVMPLAT_ASINCLUDES-y        += -I$(UK_PLAT_COMMON_BASE)/include
 LIBKVMPLAT_CINCLUDES-y         += -I$(LIBKVMPLAT_BASE)/include
 LIBKVMPLAT_CINCLUDES-y         += -I$(UK_PLAT_COMMON_BASE)/include
 LIBKVMPLAT_CINCLUDES-y         += -I$(UK_PLAT_DRIVERS_BASE)/include
+LIBKVMPLAT_CINCLUDES-y         += -I$(CONFIG_UK_BASE)/lib/ukofw/include
```

Comment:
This one should go away. To make the headers of `libukofw` publicly available what you need to do instead is define the following in `lib/ukofw/Makefile.uk`:
```
ASINCLUDES-$(CONFIG_LIBUKOFW)         += -I$(LIBUKOFW_BASE)/include
CINCLUDES-$(CONFIG_LIBUKOFW)          += -I$(LIBUKOFW_BASE)/include
CXXINCLUDES-$(CONFIG_LIBUKOFW)        += -I$(LIBUKOFW_BASE)/include
```

Notice the diference between `CINCLUDES` (public) and `LIBUKOFW_CINCLUDES` (headers available internally to the library).